query/query4_test.go

```diff
@@ -1564,3 +1564,51 @@ func TestNumUids(t *testing.T) {
 	require.Equal(t, metrics.NumUids["name"], uint64(16))
 	require.Equal(t, metrics.NumUids["_total"], uint64(26))
 }
+
+func TestJaegerTag(t *testing.T) {
+	query := `
+		 {
+			 #tag:test
+			 me(func: uid(0x01)) {
+				 uid
+			 }
+		 }
+	 `
+	js := processQueryNoErr(t, query)
+	require.JSONEq(t,
+		`{"data": {"me":[ {"uid": "0x1"}]}}`,
+		js)
+}
+
+func TestJaegerTag1(t *testing.T) {
+	query := `
+		 {
+			 me(func: uid(0x01)) {
+				 uid
+				 #tag:test
```

Comment:
@MichelDiz will this tag the query in jager? Is it possible to check for that?

Reply:
Yes, I carried out all these tests while running Jaeger concurrently. They all succeeded.

It would be conceivable to think of a process that could somehow infer this from Jaeger's API. However, I did not delve into this due to its complexity. Therefore, I focused on the requests, aiming to reproduce panics with the tags, which in this case, occurred without any panics.

Reply:
But that is not right. Isn't this a `#tag:test` predicate in this case? It shouldn't be interpreted as a jaeger tag from what I understand.

Reply:
Nope, it is ignored by the parse as it is a comment. You can comment anywhere in the query. As long it doesn't break the syntax.

If you check the code. What I'm doing is just inferring comments. With that I can capture the tag. Because Dgraph passes the complete query at this stage of execution. This step is before parsing. Parsing happens after that stage.